appveyor.yml

```diff
@@ -9,6 +9,9 @@ environment:
     - nodejs_version: "4"
     - nodejs_version: "5"
     - nodejs_version: "6"
+    - nodejs_version: "8"
+    - nodejs_version: "10"
+    - nodejs_version: "12"
```

Comment:
This might error if appveyor hasn't fixed a bug they were having over the weekend where 12 wouldn't be pulled down 😦

Reply:
Looks like it's still a bug.  Can you comment it out with a note?